drivers/network/dd/pcnet/netpcnet.inf

```diff
@@ -20,7 +20,7 @@ DefaultDestDir = 12
 [AMDMfg]
 %PCNET.DeviceDesc% = PCNet_Inst.ndi,PCI\VEN_1022&DEV_2000
 
-;----------------------------- PCNET DRIVER -----------------------------
+;----------------------------- PCNET DRIVER ---------------------------
```

Comment:
What problem is this solving, other than to make the right side shorten than the left side for no good reason?

Reply:
If you watch the individual commits and their descroption, you will see from the description that it harmonizes the length of that line to the other separator lines within this file.